app/src/main/kotlin/com/wire/android/ui/settings/devices/e2ei/E2eiCertificateDetailsScreenNavArgs.kt

```diff
@@ -18,5 +18,7 @@
 package com.wire.android.ui.settings.devices.e2ei
 
 data class E2eiCertificateDetailsScreenNavArgs(
-    val certificateString: String
+    val certificateString: String,
+    val isSelfUser: Boolean,
+    val otherUserName: String? = null
```

Comment:
this feature is fancy and no requirements on it, and display name on the file might be more readable, didn't want to do a query to fetch data that is unnecessary, also since we add a date to the end of the file, duplicate names won't occur.